marketplace/core/types/ecommerce/vtex/tests/test_views.py

```diff
@@ -470,3 +485,90 @@ def view(self):
         Returns the view that will be used in the test.
         """
         return self.view_class.as_view()
+
+
+class LinkCatalogTestCase(SetUpService):
+    def setUp(self):
+        super().setUp()
+        self.app = apptype.create_app(
+            created_by=self.user, project_uuid=str(uuid.uuid4())
+        )
+        self.wpp_cloud_app = App.objects.create(
+            code="wpp-cloud", created_by=self.user, project_uuid=self.app.project_uuid
+        )
+
+        self.user_authorization = self.user.authorizations.create(
+            project_uuid=self.app.project_uuid
+        )
+        self.user_authorization.set_role(ProjectAuthorization.ROLE_ADMIN)
+
+        self.url = reverse("vtex-app-link-catalog", kwargs={"uuid": self.app.uuid})
+        self.body = {
+            "catalog_id": "catalog123",
+            "domain": "test.vtex.com",
+            "store_domain": "store.vtex.com",
+            "app_key": "test_app_key",
+            "app_token": "test_app_token",
+            "wpp_cloud_uuid": str(self.wpp_cloud_app.uuid),
+            "project_uuid": self.app.project_uuid,
+        }
+
+    @property
+    def view(self):
+        return self.view_class.as_view({"post": "link_catalog"})
+
+    @patch(
+        "marketplace.core.types.ecommerce.vtex.usecases.link_catalog_start_sync.LinkCatalogAndStartSyncUseCase.link_catalog",  # noqa: E501
+        return_value=True,
+    )
+    def test_link_catalog_success(self, mock_link_catalog):
+        # Fazer a requisição
```

Comment:
Just a word of warning: to maintain consistency throughout the project, we usually write all comments and documentation in English. This helps everyone on the team (and future contributors) better understand the codebase. Could you translate these comments? 😊